iothub/service/src/Jobs/ScheduledTwinUpdate.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Contains fields used when creating a job to update twin tags and desired properties on one or multiple devices.
+    /// </summary>
+    public class ScheduledTwinUpdate
+    {
+        /// <summary>
+        /// Query condition to evaluate which devices to run the job on.
+        /// </summary>
+        [JsonProperty(PropertyName = "queryCondition", Required = Required.Always)]
+        public string queryCondition { get; set; }
+
+        /// <summary>
+        /// Twin object to use for the update
```

Comment:
All of these should have ending punctuation.

```suggestion
        /// Twin object to use for the update.
```